Implement systray integration

mpotthoff opened the pull request:
I'm using this a lot and actually had the same problems as others in #16. So I just implemented systray support myself. I'm open for any improvements you might have. I could also hide it behind a `--systray` flag if you want.

benpye commented:
I should really turn off that AppVeyor build now 🙂

If you could put this behind a `--systray` flag that would be great.

benpye commented:
Another comment, is checking in the icon.go the correct way to go here - I haven't tried but is there a way to tie this together with go generate so we just keep the source icon file?

mpotthoff commented:
Okay so I added a systray flag and switched to go generate. Oh and I also added some documentation on how to build ;) Anything else that you would like to see changed?

benpye commented:
Merged 👍

schrej commented on Readme.md:
````diff
@@ -42,6 +46,17 @@ Grab the latest release on the [releases page](https://github.com/benpye/wsl-ssh
+
+To create a build without a console window:
+```
+go build -ldflags -H=windowsgui
````
I guess the release doesn't include this?
